va/va.h

```diff
@@ -2350,6 +2351,8 @@ typedef struct _VAPictureParameterBufferH264
         unsigned int value;
     } pic_fields;
     unsigned short frame_num;
+    unsigned char num_ref_idx_l0_default_active_minus1;
+    unsigned char num_ref_idx_l1_default_active_minus1;
```

Comment:
@fhvwy I don't have a working implementation or documentation so far. 

I think we can add some additional reserved bytes here. Once we have a workable driver with the new feature, we can reuse these reserved bytes and ABI won't be broken.